src/Sepes.Infrastructure/Service/Azure/AzureCostManagementService.cs

```diff
@@ -30,6 +30,22 @@ public async Task<double> GetVmPrice(string region, string size, CancellationTok
             }
 
             return relevantPriceItem.retailPrice * 730; //Prices are per hour, azure defaults to 730 hours/month in their web interface
-        }       
+        }
+
+        public async Task<double> GetSizePrice(string size, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            var priceUrl = $"https://prices.azure.com/api/retail/prices?$filter=serviceName eq 'Virtual Machines' and armSkuName eq '{size}' and priceType eq 'Consumption'";
+
+            var prices = await GetResponse<AzurePriceResponseDto>(priceUrl, false, cancellationToken);
+
+            var relevantPriceItem = prices.Items.Where(p => p.effectiveStartDate <= DateTime.UtcNow).OrderByDescending(p => p.retailPrice).FirstOrDefault();
+
+            if (relevantPriceItem == null)
+            {
+                return 0.0;
+            }
+
+            return relevantPriceItem.retailPrice * 730; //Prices are per hour, azure defaults to 730 hours/month in their web interface
+        }
```

Comment:
I made an extra function for get size. Are you okay with this? It is the same code except for not including region in the request, So can probably refactor them into one or something